creator-node/src/config.js

```diff
@@ -562,10 +562,10 @@ const config = convict({
     env: 'minimumSecondaryUserSyncSuccessPercent',
     default: 50
   },
-  maxNumberHoursPrimaryRemainsUnhealthy: {
-    doc: 'The max number of hours since first failed health check that a primary can still be marked as healthy',
+  maxNumberSecondsPrimaryRemainsUnhealthy: {
+    doc: 'The max number of seconds since first failed health check that a primary can still be marked as healthy',
     format: 'nat',
-    env: 'maxNumberHoursPrimaryRemainsUnhealthy',
+    env: 'maxNumberSecondsPrimaryRemainsUnhealthy',
     default: 24
```

Comment:
can't be 24 anymore, needs to be 86400

Reply:
also i think we need to specify a more sensible value for local dev as well? otherwise can't test this flow in maddog /  manually right

Reply:
oh whoops,updating

Reply:
yes you are correct

Reply:
im going to set the primary unhealthy threshold to 5s for local dev